src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommandHandler.cs

```diff
@@ -111,27 +103,16 @@ private async Task SendNotification(CommitmentView commitment, SubmitCommitmentC
                     break;
             }
 
-            foreach (var email in emails)
+            var emailMessage = new EmailMessage
             {
-                _logger.Info($"Sending email to {email}");
-                await _mediator.SendAsync(BuildNotificationCommand(email, templateId, tokens));
-            }
-        }
-
-        private SendNotificationCommand BuildNotificationCommand(string email, string templateId, Dictionary<string, string> tokens)
-        {
-            return new SendNotificationCommand
-            {
-                Email = new Email
-                {
-                    RecipientsAddress = email,
-                    TemplateId = templateId,
-                    ReplyToAddress = "noreply@sfa.gov.uk",
-                    Subject = "x",
-                    SystemId = "x",
-                    Tokens = tokens
-                }
+                TemplateId = templateId,
+                Tokens = tokens
             };
+
+            await _providerEmailService.SendEmailToAllProviderRecipients(
+                commitment.ProviderId.GetValueOrDefault(),
```

Comment:
No doubt you inherited this, but it seems odd that providerId is nullable

Reply:
This is defined on an object on the commitments api, feels like changing that is outside of the scope of this story?